src/styles/index.scss

```diff
@@ -1,3 +1,48 @@
+@import './variables';
+@import './card';
+@import './stars';
+@import './header';
+
+@font-face {
+  font-family: Roboto;
+  src: url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  min-height: 100vh;
+}
+
 body {
   margin: 0;
 }
+
+.catalog {
+  display: grid;
+  grid-auto-rows: minmax(auto, max-content);
+  grid-template-columns: repeat(1, $grid-size);
```

Comment:
The grid-template-columns property uses $grid-size. Ensure that $grid-size is defined in your variables file and is appropriate for the layout you intend to achieve.